cmake/ProjectDependenciesSetup.cmake

```diff
@@ -0,0 +1,40 @@
+function(disable_if_no_fortran package)
```

Comment:
Is this somehow now pulled into the CMakeLists.txt from where it was removed?

Reply:
See https://github.com/gsjaardema/seacas/pull/214#issuecomment-776169266